src/win32.c

```diff
@@ -1032,4 +1032,35 @@ gchar *win32_get_user_config_dir(void)
 	return g_build_filename(g_get_user_config_dir(), "geany", NULL);
 }
 
+
+/* Retrieve the console codepage
+ * In case GetConsoleOutputCP() returns 0 (i.e. the application doesn't have an own console window
+ * fallback to GetOEMCP(). */
+guint win32_get_console_codepage(void)
+{
+	guint codepage = GetConsoleOutputCP();
```

Comment:
I don't know either, but there seem to be 2 different functions, and I would imagine a script is an input rather than an output… but I have no idea really.